tests/contracts/nonfungible-token/BoolUtils.scilla

```diff
@@ -0,0 +1,30 @@
+library BoolUtils
```

Comment:
`BoolUtils` is already part of `stdlib`. So why is this added again? Just a plain `import BoolUtils` in the Scilla file should suffice.